packages/react-native-bridge/android/settings.gradle

```diff
@@ -1,5 +1,52 @@
-rootProject.name = '@wordpress/react-native-bridge'
+pluginManagement {
+  plugins {
+    id 'com.jfrog.bintray' version '1.8.5'
+  }
+}
 
+rootProject.name = '@wordpress_react-native-bridge'
+
+include ':react-native-bridge'
 include ':@wordpress_react-native-aztec'
 project(':@wordpress_react-native-aztec').projectDir = new File(rootProject.projectDir, '../../react-native-aztec/android')
 
+def publishReactNativeBridgeVersion = properties["publishReactNativeBridgeVersion"]
+gradle.ext.willPublishReactNativeBridgeBinary = publishReactNativeBridgeVersion?.trim() as boolean
+
+// Only add these modules/projects if we are not building to publish a binary.
+if (!gradle.ext.willPublishReactNativeBridgeBinary) {
+    /*
+    IMPORTANT: DO NOT add hermes modules if we are building a binary to publish.
+    If these modules are added, it'll cause the `com.facebook.hermes.BuildConfig` file to be
+    included in the binary which will then result in a duplicate class error.
+    This seems to happen even if these modules are not added as a dependency.
+    */
```

Comment:
Thank you for explaining _why_ we shouldn't add Hermes when not building for publishing. 👍